2_observations.yml

```diff
@@ -32,11 +37,17 @@ targets:
   ngwos_sites:
     command: filter_ngwos(ngwos_ind = '2_observations/out/NGWOS_sites_data.rds.ind')
 
+  2_observations/out/extra_flow_sites.rds.ind:
+    command: get_flow_sites(
+      flow_ind = '2_observations/in/daily_flow.rds.ind',
+      temp_sites_ind = '2_observations/in/all_sites.rds.ind',
```

Comment:
where do `daily_flow.rds.ind` and `all_sites.rds.ind` come from? do they come from the national pipeline? If so, it'd probably good to note that somewhere. Is it enough just to write in a comment above this target?